src/modules/peek/Peek.UI/PeekXAML/Views/TitleBar.xaml

```diff
@@ -30,8 +36,7 @@
                 Width="24"
```

Comment:
I believe we use `16` in other apps. Shouldn't we do that here?

Reply:
True. Will do.